### PR TITLE
Normalize the package name for Doc.Ms readme

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -235,4 +235,9 @@ function GetExistingPackageVersions ($PackageName, $GroupId=$null)
     LogError "Failed to retrieve package versions. `n$_"
     return $null
   }
+}
+
+# Turn the package name start with `Azure.Identity` to "Identity".
+function Normalize-dotnet-Package-name ($PackageName) {
+  return $PackageName -replace "Azure." , ""
```
Comment (chidozieononiwu) [inline eng/scripts/Language-Settings.ps1:242]: Will this ever be needed for Track 1 libraries? How about management? Seem it won't work for those.

File: eng/scripts/Language-Settings.ps1 [hunk 2]
```diff
@@ -64,6 +64,7 @@ function Get-dotnet-PackageInfoFromPackageFile ($pkg, $workingDirectory)
   Expand-Archive -Path $zipFileLocation -DestinationPath $workFolder
   [xml] $packageXML = Get-ChildItem -Path "$workFolder/*.nuspec" | Get-Content
   $pkgId = $packageXML.package.metadata.id
+  $docsReadMeName = $pkgId -replace "Azure." , ""
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:67]:
```suggestion
  $docsReadMeName = $pkgId -replace "^Azure." , ""
```

**Review by weshaggard — APPROVED** [review]

File: eng/scripts/Language-Settings.ps1 (hunk 1 above)
Comment (weshaggard) [reply eng/scripts/Language-Settings.ps1]: I suspect we will strip on Azure from the front of the track 2 management libraries as well.

Comment (sima-zhu) [reply eng/scripts/Language-Settings.ps1]: We keep the one not start with Azure as it is for Doc.Ms readme file. So the change here does not apply to track 1 and management.

**Pull request merged**